pytorch_lightning/utilities/imports.py

```diff
@@ -92,4 +93,4 @@ def _compare_version(package: str, op, version) -> bool:
 
 from pytorch_lightning.utilities.xla_device import XLADeviceUtils  # noqa: E402
 
-_TPU_AVAILABLE = XLADeviceUtils.tpu_device_exists()
+_TPU_AVAILABLE = XLADeviceUtils.tpu_device_exists() or int(os.getenv("PL_TPU_AVAILABLE", 0))
```

Comment:
Why is this required in this PR?

Reply:
Trying to mock TPU, but this doesn't work.

Reply:
then pls rever it